poke/src/components/Card.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+    <div class="card">
+        <div class="title">
+           
+            <slot name="title" />
+        </div>
+
+        <div class="content">
+          <slot name="content" />
+         
+        </div>
+        
+        <div class="description">
+          <slot name="description"/>
+
+        </div>
+    </div>
+</template>
+<script>
+export default {
```

Comment:
where is the PokemonCards component? @Priya-Sandal  this step is missing

```
Create a PokemonCards.vue component, it will be the parent component for the card component so our app.vue looks nice and clean. 
So app.vue now has logic for API calls. Then render the PokemonCards component, and send pokemon data to it.
The PokemonCards.vue component will take the data from app.vue as props and render card.vue component to show the pokemon cards.
```